common/content/statusline.js

```diff
@@ -148,13 +148,14 @@ const StatusLine = Module("statusline", {
         var toggle_off = function () {
             bb.style.height = '0px';
             bb.style.overflow = 'hidden';
+            sv.contentSeparator = highlight.get('ContentSeparator').value;
```

Comment:
Correct. `setVisibility()` seems to be called very early, such that line 133 is executed even before the `.vimperatorrc` is loaded (which loads the colorscheme). Of course, when you later call `:highlight ContentSeparator` it won't be updated as well. Thus it contains only the default value.